includes/Export.php

```diff
@@ -965,22 +1203,31 @@ function write( $string ) {
  * @ingroup Dump
  */
 class Dump7ZipOutput extends DumpPipeOutput {
+	/**
+	 * @param string $file
+	 */
 	function __construct( $file ) {
 		$command = $this->setup7zCommand( $file );
 		parent::__construct( $command );
 		$this->filename = $file;
 	}
 
+	/**
+	 * @param string $file
+	 * @return string
+	 */
 	function setup7zCommand( $file ) {
-		// Wikia change - begin - @upstream: I07ab5f93ecd6d706460691db5181de89ef31cbea
 		$command = "7za a -bd -si -mx=4 " . wfEscapeShellArg( $file );
-		// Wikia change - end
 		// Suppress annoying useless crap from p7zip
 		// Unfortunately this could suppress real error messages too
 		$command .= ' >' . wfGetNull() . ' 2>&1';
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$command". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)